data/json/items/tool/explosives.json

```diff
@@ -1615,5 +1615,62 @@
       "explosion": { "power": 69200, "shrapnel": { "casing_mass": 12000, "fragment_mass": 600 } }
     },
     "flags": [ "BOMB", "TRADER_AVOID", "DANGEROUS" ]
+  },
+  {
+    "id": "jug_bomb",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "jug bomb" },
+    "description": "A glass jug with gasoline spilled on the inside and a small pouch of oxidzier powder next to it, upon mixing these creat a incredibly shock explosives bomb, when activited you mix the two ingreidents its best to throw it as it might go off in your hands!",
```

Comment:
```suggestion
    "description": "A glass jug with gasoline spilled on the inside and a small pouch of oxidizer powder next to it, which upon mixing will create an incredible shock explosion. Activating this will mix the two ingredients, after which it's best to throw it as it might go off in your hands!",
```

Reply:
I called it a jug as in the source material, a sling ready version might be feasible but I would imagine it to be dangerous and unstable, maybe you could make it safe for sling by taping the oxy powder to the roof of the jar to make the 2 ingredients mix upon the end whipping motion but I am not certain so  I decided to not add it